### PR TITLE
CurveBS: curve_ops_tool show space usage of every logicalpool

### PR DESCRIPTION
**Pull request by yuxubinchen**
Signed-off-by: Zhao ZeYu <2302370185@qq.com>

### What problem does this PR solve?

Issue Number: #1055  

Problem Summary: curve_ops_tool show space usage of every logicalpool

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
Can the results of the test be pasted here?

**Comment by Cyber-SiKu** [comment]
Can you show some output?

File: src/tools/status_tool.cpp [hunk 1]
```diff
@@ -162,7 +162,8 @@ void StatusTool::PrintHelp(const std::string& cmd) {
 
 int StatusTool::SpaceCmd() {
     SpaceInfo spaceInfo;
-    int res = GetSpaceInfo(&spaceInfo);
+    std::vector<LogicalpoolSpaceInfo> lpoolspaceinfo;
```
Comment (Cyber-SiKu) [inline src/tools/status_tool.cpp:165]: lpoolspaceinfo can be a member of SpaceInfo.IMO:map or unordered_map is better, but there is nothing wrong with vector.

**Comment by yuxubinchen** [comment]
recheck

**Comment by yuxubinchen** [comment]
recheck

**Review by Cyber-SiKu — APPROVED** [review]
LGTM

File: src/tools/status_tool.h [hunk 2]
```diff
@@ -58,28 +59,28 @@ using curve::mds::topology::ChunkServerInfo;
...
     uint64_t usedChunkSize = 0;
     // 总体能容纳的文件大小
     uint64_t totalCapacity = 0;
     // 分配大小
     uint64_t allocatedSize = 0;
+    // recycleBin的分配大小
+    uint64_t recycleAllocSize = 0;
+    // 系统中存在的文件大小
+    uint64_t currentFileSize = 0;
+    std::unordered_map<u_int32_t, LogicalpoolSpaceInfo> lpoolspaceinfo;
```
Comment (Cyber-SiKu) [inline src/tools/status_tool.h:83]: uint32_t

Comment (yuxubinchen) [reply src/tools/status_tool.h]: fixed

**Comment by yuxubinchen** [comment]
recheck

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**